packages/now-python/src/index.ts

```diff
@@ -115,7 +136,10 @@ export const build = async ({
   }
 
   console.log('Installing dependencies...');
-  await pipInstall(pipPath, workPath, 'werkzeug');
+
+  if (!meta.isDev || !(await isInstalled('werkzeug'))) {
```

Comment:
What's the purpose of checking for `werkzeug` here? Isn't `isDev` sufficient?

Reply:
probably new users won’t have werkzeug installed, so install it when running in now dev and not present